app/assets/stylesheets/course/forum/posts.scss

```diff
@@ -0,0 +1,5 @@
+.course-forum-posts {
+  &.reply {
+    @include forum-post;
```

Comment:
OK, I understand you. I'll just keep the common part in this mixin and move others to topics.scss